jck/jck.mk

```diff
@@ -60,7 +60,17 @@ endif
 OTHER_OPTS=
 # if JDK_IMPL is openj9 or ibm
 ifneq ($(filter openj9 ibm, $(JDK_IMPL)),)
- OTHER_OPTS=-Xtrace:maximal=all{level2}
+  String sectionHeaderHelpTextStyleCss = ' color: #444; font-family: Roboto, sans-serif !important; outline: 0; display: block; font-weight: bold; padding-left: 5px; margin-bottom: 1rem'
+ String separatorHelpTextStyleCss = ' border: none; margin-top: 0; margin-bottom: 0 '
+ARCH_OS_LIST.each { ARCH_OS ->
+	JDK_VERSIONS.each { JDK_VERSION ->
+		LEVELS.each { LEVEL ->
+			GROUPS.each { GROUP ->
+				def ACTUAL_TEST_JOB_NAME = TEST_JOB_NAME ? TEST_JOB_NAME : "Test_openjdk${JDK_VERSION}_${JDK_IMPL_SN}_${LEVEL}.${GROUP}_${ARCH_OS}${SUFFIX}"
+				def ACTUAL_TARGET = TARGET ? TARGET : "${LEVEL}.${GROUP}"
+				def ACTUAL_BUILD_LIST = BUILD_LIST ? BUILD_LIST : GROUP
+
+
```

Comment:
Not really. I posted the code in https://github.com/adoptium/aqa-tests/issues/1754#issuecomment-932460305. We need to add the code here.

Reply:
Oh okay